hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -26,6 +26,7 @@
     - name: Load parameters
       ansible.builtin.include_vars:
         dir: "{{ item }}"
+        ignore_unknown_extensions: true
```

Comment:
lgtm, but should we add `extensions: ` instead to make it more controllable?

Reply:
I had that in one of the patches of the base change. I'll read it to support YAML only, it's true that we usually don't load json that's in the list by default so I'm fine.